budgetkey_data_pipelines/pipelines/procurement/tenders/parse_page_data.py

```diff
@@ -5,7 +5,7 @@
 from datetime import datetime
 
 TABLE_SCHEMA = {
-    "primaryKey": ["publication_id"],
+    "primaryKey": ["publication_id", "tender_type", "tender_id"],
```

Comment:
The reason why I removed tender id was that it sometimes got a `null` value (thus failing the processor).

This was the problematic tender: 
https://www.mr.gov.il/officestenders/Pages/officetender.aspx?pID=596701

Reply:
Perhaps "publication_id" & "tender_type" is enough?

Reply:
many central tenders have publication_id = 0 - and are identified by tender_id instead

Reply:
regarding the null value - looks like it was fixed